cmd/wharf/run.go

```diff
@@ -92,6 +90,7 @@ https://iver-wharf.github.io/#/usage-wharfyml/`,
 				BuildOptions: worker.BuildOptions{
 					StageFilter: runFlags.stage,
 				},
+				Config:        rootConfig,
 				CurrentDir:    currentDir,
 				Namespace:     ns,
```

Comment:
You're passing both the root config AND the namespace. Suggest to only use the namespace from the rootConfig.K8s.Namespace that's already populated loadKubeconfig

Reply:
Removed namespace and split to only use the required configs `rootConfig.K8s` & `rootConfig.Worker`